Sample Applications/WPFGallery/Controls/ControlExample.xaml

```diff
@@ -70,18 +70,18 @@
 
                                         <Button Grid.Column="1" Padding="8" Command="ApplicationCommands.Copy" CommandParameter="Copy_XamlCode" FocusManager.IsFocusScope="True" >
                                           <Grid>
-                                            <TextBlock x:Name="CopyGlyph_TextBlock" FontFamily="Segoe Fluent Icons" Text="&#xE8C8;"/>
-                                            <TextBlock x:Name="SuccessGlyph_TextBlock" FontFamily="Segoe Fluent Icons" Text="&#xE73E;" Opacity="0" />
+                                            <TextBlock x:Name="CopyGlyphTextBlock" FontFamily="Segoe Fluent Icons" Text="&#xE8C8;"/>
```

Comment:
Here rather than changing the name can we fix this problem by making the TextBlock non-focusable ?

Reply:
@dipeshmsft - This will not directly address the issue of characters in the Name property but will ensure that the element cannot be focused on. However we can combine both solutions, cleaning up the Name property and marking TextBlock as non-focusable for this scenario.

Reply:
Let's keep setting non-focusable aside. I am good with the PR.